de.dlr.sc.overtarget.language/model/generated/Overtarget.genmodel

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <genmodel:GenModel xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore"
-    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" copyrightText="******************************************************************************&#xA;Copyright (c) 2018-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.&#xA; *&#xA;This program and the accompanying materials are made available under the&#xA;terms of the Eclipse Public License 2.0 which is available at&#xA;http://www.eclipse.org/legal/epl-2.0.&#xA; *&#xA;SPDX-License-Identifier: EPL-2.0&#xA; ******************************************************************************"
+    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" copyrightText="Copyright (c) 2018-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.&#xA;&#xA;This program and the accompanying materials are made available under the&#xA;terms of the Eclipse Public License 2.0 which is available at&#xA;http://www.eclipse.org/legal/epl-2.0.&#xA;&#xA;SPDX-License-Identifier: EPL-2.0"
     modelDirectory="/de.dlr.sc.overtarget.language/src-gen" modelPluginID="de.dlr.sc.overtarget.language"
     forceOverwrite="true" modelName="Overtarget" updateClasspath="false" rootExtendsClass="org.eclipse.emf.ecore.impl.MinimalEObjectImpl$Container"
-    complianceLevel="6.0" copyrightFields="false" runtimeVersion="2.18">
+    complianceLevel="8.0" copyrightFields="false" runtimeVersion="2.20">

```

Comment:
Did you change that on purpose?

Reply:
No, it was changed when I generated the xtext language.